crates/solver/src/settlement_ranker.rs

```diff
@@ -151,3 +152,67 @@ impl SettlementRanker {
         Ok((rated_settlements, errors))
     }
 }
+
+fn compare_solutions(lhs: &RatedSettlement, rhs: &RatedSettlement, decimals: i32) -> Ordering {
```

Comment:
Decimals can be unsigned now, right? Because only positive values make sense. Noticed this because the test is still using negative values which doesn't break the function but also does nothing.

Reply:
Technically you could still go negative and compare fractions of WEI with one another. -2 would mean 0.12 and 0.16 would still be different. We likely don't need it so happy to turn this unsigned.